### PR TITLE
Pullrequest notification channels

### PR DESCRIPTION
**Pull request by BenSartor**

### CODE REVIEW

File: app/src/main/java/org/simlar/service/NotificationChannelId.java [hunk 1]
```diff
@@ -21,8 +21,44 @@
...
+	}
+
+	public static void createNotificationChannels(final Context context)
+	{
+		if (Build.VERSION.SDK_INT < Build.VERSION_CODES.O) {
+			return;
+		}
+
+		final NotificationManager notificationManager = Util.getSystemService(context, Context.NOTIFICATION_SERVICE);
+		for(final NotificationChannelId value: values()) {
```
Comment (jritzerfeld) [inline app/src/main/java/org/simlar/service/NotificationChannelId.java]: Format source, please: `for (`

Comment (BenSartor) [reply app/src/main/java/org/simlar/service/NotificationChannelId.java]: Thanks! See 5e5dd769aef3b2c4f8f67747297dcbe940d6af66

File: app/src/main/java/org/simlar/helper/NotificationChannelId.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ */
+
+package org.simlar.helper;
+
+public enum NotificationChannelId
```
Comment (jritzerfeld) [inline app/src/main/java/org/simlar/helper/NotificationChannelId.java]: Maybe `SimlarNotificationChannelId`?

File: app/src/main/java/org/simlar/service/NotificationChannelId.java [hunk 3]
```diff
@@ -33,7 +33,7 @@
 
 public enum NotificationChannelId
 {
-	INCOMING_CALL(NotificationManagerCompat.IMPORTANCE_LOW, R.string.app_name),
+	INCOMING_CALL(NotificationManagerCompat.IMPORTANCE_LOW, R.string.notification_channel_call_name),
```
Comment (jritzerfeld) [inline app/src/main/java/org/simlar/service/NotificationChannelId.java]: `NotificationManager.IMPORTANCE_LOW`?

File: app/src/main/java/org/simlar/service/NotificationChannelId.java [hunk 4]
```diff
@@ -33,7 +33,7 @@
...
 	MISSED_CALL(NotificationManagerCompat.IMPORTANCE_DEFAULT, R.string.missed_call_notification);
```
Comment (jritzerfeld) [inline app/src/main/java/org/simlar/service/NotificationChannelId.java]: `NotificationManager.IMPORTANCE_DEFAULT`?

**Pull request merged**